### PR TITLE
docs: enable_workflow_job_labels_check -> enable_runner_workflow_job_labels_check_all

### PR DESCRIPTION
**Pull request by NadavOps**
enable_workflow_job_labels_check is no longer exist, correct?
and now enable_runner_workflow_job_labels_check_all is the functioning variable, correct?

This is just to swap in the READMEs and samples

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1160,7 +1160,7 @@ See the [GitHub release](https://github.com/philips-labs/terraform-aws-github-ru
 
 ### Features
 
-* Strict label check and replace disable_check_wokflow_job_labels by opt in enable_workflow_job_labels_check ([#1591](https://github.com/philips-labs/terraform-aws-github-runner/issues/1591)) ([405b11d](https://github.com/philips-labs/terraform-aws-github-runner/commit/405b11db828234bfb1eb8482493a25505ce59a34))
+* Strict label check and replace disable_check_wokflow_job_labels by opt in enable_runner_workflow_job_labels_check_all ([#1591](https://github.com/philips-labs/terraform-aws-github-runner/issues/1591)) ([405b11d](https://github.com/philips-labs/terraform-aws-github-runner/commit/405b11db828234bfb1eb8482493a25505ce59a34))
```
Comment (npalm) [inline CHANGELOG.md:1163]: Can you revert the changelog, the log is generated based on the PRs
```suggestion
```

**Review by npalm — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NadavOps) [reply CHANGELOG.md]: Sure. Removed `CHANGELOG` changes

**Review by npalm — DISMISSED** [review]

**Comment by sdarwin** [comment]
With the PR, `enable_runner_workflow_job_labels_check_all` is being mentioned in the multi-runner README.md and variables.tf.

However, `enable_runner_workflow_job_labels_check_all`  doesn't actually exist in the context of multi-runner.   To check that:  

```

cd terraform-aws-github-runner
grep -r "enable_runner_workflow_job_labels_check_all"
```

Its only appearance is in the root level variables.tf and main.tf.  Do those apply in /examples/multi-runner/ or modules/multi-runner/?  Correct me if this is wrong.  I think what should happen, is to slightly refactor the code of multi-runner so that it uses the variable   `enable_runner_workflow_job_labels_check_all` instead of exactmatch.

**Comment by npalm** [comment]
@sdarwin I hope to have tomorrow some time to dig in the label part. It certainly require some owrk. And evenutally I think some reafactoring.

**Comment by NadavOps** [comment]
Did I fat finger close the PR? @npalm

**Comment by NadavOps** [comment]
BTW it seems it is mentioned in variables only in the description as it actually take it from another place @sdarwin

**Review by npalm — APPROVED** [review]
LGTM, thx!

**Comment by sdarwin** [comment]
> it actually take it from another place @sdarwin

@npalm which place?

The variable "enable_runner_workflow_job_labels_check_all" is defined in the root level, top level, files `variables.tf `and `main.tf`.   

https://github.com/philips-labs/terraform-aws-github-runner/blob/main/main.tf
https://github.com/philips-labs/terraform-aws-github-runner/blob/main/variables.tf

Does multi-runner reference those files?

**Comment by sdarwin** [comment]
There are two possibilities:

1. A "minimal fix".  The multi-runner example files should be configured to use  "enable_runner_workflow_job_labels_check_all".   Currently https://github.com/philips-labs/terraform-aws-github-runner/tree/main/examples/multi-runner doesn't mention that variable.  

2. A more in-depth fix as mentioned in my earlier comments.

**Pull request merged**